core/src/test/java/org/apache/druid/data/input/impl/MapInputRowParserTest.java

```diff
@@ -31,28 +30,27 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
-import java.util.List;
-import java.util.Set;
-
 public class MapInputRowParserTest
 {
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
   private final TimestampSpec timestampSpec = new TimestampSpec("time", null, null);
-  private final List<String> dimensions = ImmutableList.of("dim");
-  private final Set<String> dimensionExclusions = ImmutableSet.of();
```

Comment:
The behavior hasn't changed. I made the test more realistic since the timestamp field name is always in `dimensionExclusions` in production. See https://github.com/apache/druid/blob/master/server/src/main/java/org/apache/druid/segment/indexing/DataSchema.java#L162-L177.